packages/components/Button/src/PrimaryButton/PrimaryButton.settings.win32.ts

```diff
@@ -0,0 +1,50 @@
+import { IButtonType } from '../Button.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+
+export const settings: IComposeSettings<IButtonType> = [
+  {
+    tokens: {
+      backgroundColor: 'brandBackground',
+      color: 'neutralForegroundInverted',
+      borderColor: 'brandBackground',
+    },
+    // TODO: neutralForegroundInverted is not working for icon color.
```

Comment:
nit: Leave TODO comments with the Github issue number tracking it, in this case I think it's #728 ?
But also why doesn't it work? I thought neutralForegroundInverted was a string and would work fine?

Reply:
As I understand, it uses ColorValue and it's not supported for SVGs. @rurikoaraki mentioned it in https://github.com/microsoft/fluentui-react-native/pull/906

Reply:
I think with Ruriko's change you can pass a ColorValue, and if it does not represent a string, it will downgrade to either white/black based on the theme. So in this case, If you passed neutralForegroundInverted, then:
- If neutralForegroundInverted is a string, it will render it correctly
- If not, it will show white/black based on light/dark mode. 

Could you try testing that?

Reply:
I checked. It still can't ise tokens. Colors don't change